nautobot/core/templates/generic/object_list.html

```diff
@@ -46,6 +46,61 @@
     {% endif %}
+    {% if model.is_saved_view_model %}
+        <div class="btn-group">
+            <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                <span class="mdi mdi-view-compact-outline" aria-hidden="true"></span> Saved Views <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu">
+                {% if perms.users.view_savedview %}
+                    {% for saved_view in saved_views %}
+                        <li>
+                            <a href="{{ saved_view.get_absolute_url }}">
+                                {% if saved_view == current_saved_view %}
+                                    <i class="mdi mdi-view-compact-outline text-bold" aria-hidden="true"></i>
+                                    <strong>
+                                        {{saved_view.name}}
+                                    </strong>
+                                {% else %}
+                                    <i class="mdi mdi-view-compact-outline text-muted" aria-hidden="true"></i>
+                                    {{saved_view.name}}
+                                {% endif %}
+                            </a>
+                        </li>
+                    {% endfor %}
+                    {% if saved_views %}
+                        <li role="separator" class="divider"></li>
+                    {% endif %}
+                {% endif %}
+                {% if current_saved_view %}
+                    {% if perms.users.change_savedview %}
+                        <li>
+                            <a href="{% url 'user:savedview_edit' pk=request.GET.saved_view %}{% if request.GET %}?{{ request.GET.urlencode }}{% endif %}">
+                                <i class="mdi mdi-content-save-outline text-muted" aria-hidden="true"></i>
+                                <span>Update Current View</span>
+                            </a>
+                        </li>
+                    {% endif %}
+                {% endif %}
+                {% if perms.users.add_savedview %}
+                    <li>
+                        <a href="" data-toggle="modal" data-target="#saved_view_modal" title="Save Current View As">
+                            <i class="mdi mdi-content-save-plus-outline text-muted" aria-hidden="true"></i>
+                            <span>Save As New View</span>
+                        </a>
+                    </li>
+                {% endif %}
+                {% if current_saved_view %}
+                    <li>
+                        <a href="{% url 'user:savedview' pk=request.GET.saved_view %}">
+                            <i class="mdi mdi-arrow-u-right-bottom text-danger" aria-hidden="true"></i>
+                            <span class="text-danger">Clear View</span>
```

Comment:
This, I assume, is used to undo all unsaved modifications to the view and restore the UI to its initial view state?

Reply:
If so I don't think `Clear View` might be the right wording here